manifests/charts/gateways/istio-ingress/templates/meshexpansion.yaml

```diff
@@ -16,7 +16,16 @@ spec:
         name: tcp-istiod
       hosts:
         - "*"
+    {{- if  .Values.global.centralIstiod }}
+    - port:
+        number: 8443
```

Comment:
If this works, I think its a bug in Gateway implementation.. I would expect this to be 443?

Reply:
unfortunately change this to port 443 won't work for me.  The invalid file applied fine without rejection.  Here is what didn't work:

```
$ istioctl pc listener istio-ingressgateway-64fc6959-8t99t.istio-system
ADDRESS     PORT      TYPE
0.0.0.0     15012     TCP
0.0.0.0     80        HTTP
0.0.0.0     443       TCP
0.0.0.0     15090     HTTP
(⎈ |linistio14/boft3r6w0p5blanmm18g:default)
~/go/src/istio.io/testingdir ⌚ 19:22:46
$ cat mesh-expansion443.yaml                                
apiVersion: networking.istio.io/v1alpha3
kind: Gateway
metadata:
  name: meshexpansion-gateway
  namespace: istio-system
  labels:
    release: istio
spec:
  selector:
    istio: ingressgateway
  servers:
    - port:
        number: 15012
        protocol: TCP
        name: tcp-istiod
      hosts:
        - "*"
    - port:
        number: 443
        protocol: TCP
        name: tcp-istiodwebhook
      hosts:
        - "*"
---


apiVersion: networking.istio.io/v1alpha3
kind: VirtualService
metadata:
  name: meshexpansion-vs-istiod
  namespace: istio-system
  labels:
    release: istio
spec:
  hosts:
  - istiod.istio-system.svc.cluster.local
  gateways:
  - meshexpansion-gateway
  tcp:
  - match:
    - port: 15012
    route:
    - destination:
        host: istiod.istio-system.svc.cluster.local
        port:
          number: 15012
  - match:
    - port: 443
    route:
     - destination:
        host: istiod.istio-system.svc.cluster.local
        port:
          number: 443
---


apiVersion: networking.istio.io/v1alpha3
kind: DestinationRule
metadata:
  name: meshexpansion-dr-istiod
  namespace: istio-system
  labels:
    release: istio
spec:
  host: istiod.istio-system.svc.cluster.local
  trafficPolicy:
    portLevelSettings:
    - port:
        number: 15012
      tls:
        mode: DISABLE
    - port:
        number: 443
      tls:
        mode: DISABLE
```

Reply:
This seems like the same issue as https://github.com/istio/istio/issues/23339

Reply:
yes, I come to that conclusion as well.  I 'm going to try the workaround described in #23339, e.g. change the target port from 8443 to 443 on the istio ingress gw service.

Reply:
okay that was it!!!  able to change the target port from 8443 to 443 on the istio ingress gw service and repeat the validation webhook test successfully with rejection on invalid yaml.